plugin/pkg/shoot/dns/admission_test.go

```diff
@@ -676,7 +677,7 @@ var _ = Describe("dns", func() {
 
 				err := admissionHandler.Admit(context.TODO(), attrs, nil)
 
-				Expect(err).To(Not(HaveOccurred()))
+				Expect(err).To(HaveOccurred())
```

Comment:
The test description states "should not reject shoots using a non compliant default domain on updates", but actually the opposite is expected.

Reply:
It should reject. I will update the test description